packages/typespec-ts/src/modular/helpers/operationHelpers.ts

```diff
@@ -701,13 +706,13 @@ function deserializeResponseValue(
   required: boolean
 ): string {
   const coreUtilSet = importSet.get("@azure/core-util");
+  const deserializeUtils = importSet.get("../utils/deserializeUtil.js");
```

Comment:
Not the priority, just wondering if this could work in multi-client case.

Reply:
As this deserilieUtils is highly related with models, I will consider this after #1892 is resolved.